src/stores/AllEventsStore.js

```diff
@@ -64,41 +64,11 @@ export default class {
     limit /= 2; // eslint-disable-line
     skip /= 2; // eslint-disable-line
     const orderBy = { field: 'blockNum', direction: this.app.sortBy };
-    const filters = [
-      // finalizing
-      { token: Token.Bot, status: OracleStatus.WaitResult },
-      // voting
-      { token: Token.Bot, status: OracleStatus.Voting },
-      // betting
-      { token: Token.Qtum, status: OracleStatus.Voting },
-      { token: Token.Qtum, status: OracleStatus.Created },
-      // result setting
-      { token: Token.Qtum, status: OracleStatus.OpenResultSet },
-      { token: Token.Qtum, status: OracleStatus.WaitResult },
-    ];
+    const filters = [];
```

Comment:
believe you can just pass in `undefined` for the oracle filters param since we dont have any. test it first though.